src/_data/menuItems.json

```diff
@@ -41,6 +41,10 @@
         "name": "Program",
         "hasSubmenu": true,
         "submenu": [
+            {
+                "name": "Program Guide",
+                "link": "program/guide"
```

Comment:
It's missing venue map so lemme if we wanna merge this PR and keep the page accessible at `/program/guide` URL, but hide it from the Program navbar dropdown until the map/other things are finalized. cc: @bhavin192